.github/workflows/build-release.yml

```diff
@@ -11,6 +17,16 @@ jobs:
         uses: actions/checkout@v3.1.0
       - name: Install composer dependencies
         run: composer install --no-dev -o
+      - name: Custom clean via Makefile
+        if: ${{ hashFiles('Makefile') != '' && inputs.makefile_rule != '' }}
+        run: |
+          hasCleanMakefileRule=$((cat Makefile | grep '${{ inputs.makefile_rule }}:' > /dev/null; echo $?))
+          if [ "$hasCleanMakefileRule" = "0" ]; then
+            make ${{ inputs.makefile_rule }}
+          else
+            echo No ${{ inputs.makefile_rule }} rule found in Makefile
+            exit 1
+          fi
```

Comment:
Do we need to make all these checks? `Make` will return a similar error if the rule does not exists, for instance:

```
No rule to make target 'clean'.  Stop.
```

I would keep it simple and run directly the make command, or is there any command injection you try to avoid here?

Reply:
Yeah at first I didn't think about the option input so I added more checks, but with this option it an be simplified indeed